src/oumi/core/collators/vision_language_sft_collator.py

```
@@ -126,6 +129,15 @@ def __init__(
                 - Useful for models with variable-sized outputs or heterogeneous data
                 - May be less efficient but more flexible than batch processing
                 When False (default), conversations are processed as a batch.
+
+            train_on_completions_only: If True, only compute loss on the assistant's
+                response tokens.
+
+            response_template: The template string that marks the beginning of the
+                assistant's response. Required if train_on_completions_only is True.
+
+            instruction_template: The template string that marks the beginning of the
+                user's instruction.
```

Why is the instruction template not also required like the response's?

---

We have a simpler fallback when it's not provided (masking everything except the last assistant turn)